.buildkite/bk.integration.pipeline.yml

```diff
@@ -27,6 +27,8 @@ steps:
       - integration-ess
     steps:
       - label: "Win2022:sudo:{{matrix}}"
+        depends_on:
+          - packaging-windows
```

Comment:
Just curious: how come we didn't need such explicit dependencies until now?

Reply:
Good question. The integration tests trigger step depends on the entire `int-packaging` group that wraps all the packaging steps. So these dependensies give no effect. On the other hand, they strictly outline the exact step dependencies